CHANGELOG.md

```diff
@@ -82,6 +82,8 @@ All notable, unreleased changes to this project will be documented in this file.
 - Extend `AttributeEntityType` with `CATEGORY` and `COLLECTION`. You can now assign category and collection as a attribute reference.
 - Attribute values now expose the `referencedObject`, allowing for easier access to the linked entity.
 - You can now filter and search attribute choices using the new `where` and `search` fields on the `attribute.choices` query.
+- Deprecated `Transaction.gatewayResponse` field. Please migrate to Transaction API and Apps.
```

Comment:
This could use a link to the docs for easier discovery.

Reply:
Will update in next PR to avoid rerunning the build